server/index.js

```diff
@@ -3,6 +3,7 @@ import express from 'express';
 import bodyParser from 'body-parser';
 import dotenv from 'dotenv';
 import router from './routes';
+import './database/index';
 
 dotenv.config();
 const { log } = console;
```

Comment:
What does this line of Code do? If it is redundant, i'll suggest you remove it

Reply:
> What does this line of Code do? If it is redundant, I'll suggest you remove it

Bringing in the index file where the database was initialized. The code server didn't start the database environment until I explicitly included it. Others didn't get to do that I guess. It might be removed later on to see what happens.

Reply:
ok cool